src/Sql/Sql/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 * Removed the warning messages for MSGraph migration [#18856] 
+* Move SQL Server and SQL Instance from ActiveDirectoryClient to MicrosoftGraphClient
```

Comment:
```suggestion
* Moved SQL Server and SQL Instance from ActiveDirectoryClient to MicrosoftGraphClient
```

Reply:
Thanks @BethanyZhou , Will make the change , just waiting in case someone else more suggestions. So, I can do them together.